package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-azure-functions",
-  "version": "0.7.0",
+  "version": "1.0.0-2",
```

Comment:
what's the significance of the `-2`, in the semver?

Reply:
this comes from the prerelease option, `-2` let us bump beta version until we're ready for official release.  

when we're ready, running `npm version major` will turn it into `v1.0.0`